android-appcoins-billing/src/main/java/com/appcoins/sdk/android_appcoins_billing/helpers/PayloadHelper.java

```diff
@@ -30,8 +29,8 @@
    *
    * @return The final developers payload to be sent
    */
-  public static String buildIntentPayload(@Nullable String orderReference,
-      @Nullable String developerPayload, String origin) {
+  public static String buildIntentPayload(final String orderReference,
```

Comment:
Will this always get arguments different from null now? This might require updates to documentation